hook/NewDeviceHook.js

```diff
@@ -104,15 +109,16 @@ class NewDeviceHook extends Hook {
         if(!name) {
           return // hostname is not provided by dhcp request, can't update name
         }
-
-        log.info("MAC Address", mac, " already exists, updating backup name");
-        sem.emitEvent({
-          type: "RefreshMacBackupName",
```

Comment:
This event no longer used? I see reference in `hook/RefreshMacBackupNameHook.js`

Reply:
no longer used.